newspack-theme/inc/template-functions.php

```diff
@@ -213,12 +213,17 @@ function newspack_body_classes( $classes ) {
 		$classes[] = 'feature-latest';
 	}
 
-	// Add a class when there's an ad background color.
+	// Add a class when there's an ad background color, and another when there's an ad above the footer to remove the space.
 	$ads_background_color = get_theme_mod( 'ads_color', 'default' );
+	$above_footer_ad = get_option( '_newspack_ads_placement_global_above_footer' );
```

Comment:
```suggestion
	$above_footer_ad      = method_exists( 'Newspack_Ads\Placements', 'can_display_ad_unit' ) && \Newspack_Ads\Placements::can_display_ad_unit( 'global_above_footer' );
```

The placements system provides a public method with more advanced checks to verify if the placement will render an ad unit which I believe is more suitable.

Reply:
🙌 Awesome, thanks Miguel! I made this update in [a2e5486](https://github.com/Automattic/newspack-theme/pull/1746/commits/a2e548679d56567b2ca57d2f2880abb2490dbcb8).